### ISSUE TITLE
Weird warnings/errors in webpack output

### ISSUE THREAD

**Issue by jskrzypek** [wbuchwalter/tslint-loader#71]
I'm getting a lot of these sorts warnings in my wepback output with tslint-loader@3.5.1 & tslint@5.0.0:
```
WARNING in ../ui/components/modal/components/dialog-actionsheet/dialog-actionsheet.component.ts


 @ ../ui/components/modal/index.ts 12:9-80
 @ ../app/shared/maps/maps.component.ts
 @ ../app/dashboard/menu/menu.component.ts
 @ ./module.ts
 @ ./bootstrap.ts
```
They appear to be empty. I tried reverting the changes in my PR that fixed the `NonErrorEmittedError`s from webpack, but that just wrapped the empty warnings in `NonErrorEmittedError`s...

I'm guessing this might be an interaction between tslint-loader & tslint^5? Not really sure where to start investigating.

**Comment by twjacobsen** [comment]
I see this as well. Makes it very hard to see what files actually contain warnings. Same lib versions as @jskrzypek

**Comment by Kukkimonsuta** [comment]
https://github.com/wbuchwalter/tslint-loader/pull/72

Before this gets merged you can also use it directly by referencing it in `package.json` as `'tslint-loader: 'git+https://github.com/wbuchwalter/tslint-loader.git#fix-empty-warning-reports''`

**Comment by blacksonic** [comment]
Released with 3.5.2
